core/types/subtyping.cc

```diff
@@ -14,10 +14,6 @@ TypePtr lubGround(const GlobalState &gs, const TypePtr &t1, const TypePtr &t2);
 
 TypePtr Types::any(const GlobalState &gs, const TypePtr &t1, const TypePtr &t2) {
     auto ret = lub(gs, t1, t2);
-    ENFORCE(Types::isSubType(gs, t1, ret), "\n{}\nis not a super type of\n{}\nwas lubbing with {}", ret->toString(gs),
```

Comment:
Can these use a different ENFORCE that is enabled in our test suite?

they are very useful in our test suite

Reply:
This is WIP sorry I didn't mean to request review. I just want to find the slow ones.